### PR TITLE
[Vizualization] Better error message

### PR DESCRIPTION
**Pull request by mishig25**
Better error message written by @Cadene when AV1 format is not supported on user's browser.

### CODE REVIEW

**Comment by mishig25** [comment]
cc: @Cadene @aliberts

File: lerobot/templates/visualize_dataset_template.html [hunk 1]
```diff
@@ -82,9 +82,18 @@ <h1 class="text-xl font-bold mt-4 font-mono">
             Episode {{ episode_id }}
         </h1>
 
+        <!-- Error message -->
+        <div class="font-medium text-orange-700 hidden" :class="{ 'hidden': !videoCodecError }">
+            <p>Videos could NOT play because <a href="https://en.wikipedia.org/wiki/AV1" target="_blank" class="underline">AV1</a> decoding is not available on your browser.</p>
+            <ul class="list-decimal list-inside">
+                <li>If iPhone: <span class="italic">It is supported with iPhone 15 Pro chip or higher.</span></li>
```
Comment (aliberts) [inline lerobot/templates/visualize_dataset_template.html:89]:
```suggestion
                <li>If iPhone: <span class="italic">It is supported with A17 chip or higher.</span></li>
```

File: lerobot/templates/visualize_dataset_template.html [hunk 2]
```diff
@@ -82,9 +82,18 @@ <h1 class="text-xl font-bold mt-4 font-mono">
...
+                <li>If Mac with Safari: <span class="italic">It is supported on Google Chrome or on Safari with M3 chip or higher.</span></li>
```
Comment (aliberts) [inline lerobot/templates/visualize_dataset_template.html:90]:
```suggestion
                <li>If Mac with Safari: <span class="italic">It is supported on most browsers except Safari with M1 chip or higher and on Safari with M3 chip or higher.</span></li>
```

**Review by aliberts — APPROVED** [review]
LGTM, thanks @mishig25!

Added suggestions to give more specific info for which hardware supports AV1 decoding.

**Pull request merged**